src/plone/volto/behaviors/preview_link.py

```diff
@@ -75,6 +75,8 @@ def __call__(self):
         value = self.field.get(self.context)
         if value:
             linked_image = value.to_object
+            if linked_image is None:
+                return
```

Comment:
This function is supposed to return an empty list if there's no value (see line 91).
```suggestion
                return []
```